### DISCUSSION TITLE
How to add space between last bar and price axis?

### DISCUSSION THREAD

**Discussion by vnanhtuan** [tradingview/lightweight-charts#710] [Q&A] [answered]
Hi, 
Can I add space between when init chart like image below?
![Capture1](https://user-images.githubusercontent.com/10513517/110976947-1f645680-8394-11eb-94bc-d7802bec5782.PNG)

Thank you.

**Comment by timocov** [comment, marked as answer]
See [`rightOffset`](https://github.com/tradingview/lightweight-charts/blob/master/docs/time-scale.md#time-scale-options) option.

**Reply by vnanhtuan** [reply]
Thank you
